### PR TITLE
Enable cypress test for windows and mac platform

### PR DESCRIPTION
**Pull request by heemin32**
Signed-off-by: Heemin Kim <heemin@amazon.com>

### Description
Enable cypress test for windows and mac platform

### Issues Resolved
https://github.com/opensearch-project/dashboards-maps/issues/55

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by navneet1v — COMMENTED** [review]
I can see the failing GH actions. Please fix that.

**Comment by heemin32** [comment]
I had to increase timeout values and reduce sample_geojson.json file so that cypress tests in windows to pass. 
Below screenshot shows timeout when the cypress test adds sample_geojson.json file.
Screen Shot 2022-11-23 at 1 38 42 PM

**Review by junqiu-lei — APPROVED** [review]
LGTM, thanks! I think main branch also need the change?

**Comment by heemin32** [comment]
> LGTM, thanks! I think main branch also need the change?

Yes. Will forward port it once it is approved.

File: .github/workflows/cypress-workflow.yml [hunk 1]
```diff
@@ -79,26 +116,64 @@ jobs:
...
+        run: |
+          # Resetting npm's script shell for Windows so `yarn run cypress` doesn't have conflicts
+          npm config delete script-shell
+
       - name: Run Cypress tests
         uses: cypress-io/github-action@v2
         with:
           working-directory: OpenSearch-Dashboards/plugins/custom_import_map
-          command: yarn run cypress run
+          command: yarn run cypress run --browser chrome
```
Comment (navneet1v) [inline .github/workflows/cypress-workflow.yml:162]: why we are sticking to chrome here? Should we test on multiple browsers? Does OSD has some recommendation?

Comment (heemin32) [reply .github/workflows/cypress-workflow.yml:162]: OSD uses chrome as default. https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/.github/workflows/cypress-workflow-bundle-snapshot-based-windows.yml#L31

Haven't checked the every plugins but I know some plugins also have chrome as a default browser.

**Review by navneet1v — APPROVED** [review]

**Comment by navneet1v** [comment]
Please forward port this change to the main branch.

**Pull request merged**